six/modules/python/cphd/wscript

```diff
@@ -0,0 +1,6 @@
+distclean = options = configure = lambda p: None
+
+def build(bld):
+  bld.swigModule(name = 'cphd',
+                 use = 'cphd-c++ cphd-c++ types-python mem-python six.sicd-python six-python sys-python numpyutils-c++',
+                 package = 'pysix')
```

Comment:
same remark about pep8 compliancy (2 --> 4 spaces, 72 character line limit exceeded on 'use' line, newline needed at eof, additional line needed before 'def build')

Reply:
Forgot to run `pycodestyle` on wscripts. It's compliant now